app/assets/stylesheets/admin/rebate_form.css

```diff
@@ -1,3 +1,21 @@
+/* Fonts
+=============================== */
+@font-face {
```

Comment:
These should go in a general file like `application.css` as opposed to a specific file like this one, as fonts are used globally throughout the app.

Reply:
:+1:  Will shift it